wasm/vm.go

```diff
@@ -20,6 +20,7 @@ type (
 		Globals       []uint64
 
 		OperandStack *VirtualMachineOperandStack
+		RuntimeData  interface{}
```

Comment:
could you please add the explanation of what this for?

Reply:
Added some comment.